Makefile

```diff
@@ -164,3 +171,11 @@ version:
 update-version: build
 	@echo 'Version = ${CHANGELOG_VERSION}'
 	echo ${CHANGELOG_VERSION} > .version
+
+docker-build-image:
+	docker build --build-arg GO_VERSION=$(GO_VERSION) \
+	-t cray/csi-rpm-builder .
+
+docker-rpm: docker-build-image
+	docker run --user $(CURRENT_UID):$(CURRENT_GID) --env GIT_REPO_NAME=cray-site-init --rm -v $(CURDIR):/build \
```

Comment:
I'm struggling to recall, does docker implicitly know which image to use?

Reply:
It doesn't, the next line says `cray/csi-rpm-builder` which is the image.